app/src/main/res/values/strings.xml

```diff
@@ -30,16 +30,16 @@
     <string name="action_training">Training</string>
 
     <!-- AboutActivity-->
-    <string name="version_number">Version</string>
-    <string name="about_author">Author: </string>
+    <string name="version_number" translatable="false">Version</string>
+    <string name="about_author" translatable="false">Author: </string>
```

Comment:
Adding `translatable="false"` here breaks the german translation that indeed translates this string.